### DISCUSSION TITLE
pocketbase via Java/Kotlin

### DISCUSSION THREAD

**Discussion by MHubertus** [pocketbase/pocketbase#1620] [Q&A] [answered]
Is there a neat way to use pocketbase as a backend for Java/Kotlin(sdk)?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not entirely sure that I understand what you are asking.

External non-Go apps can interact with PocketBase only via the [web APIs](https://pocketbase.io/docs/api-records/) (_there is no official Java/Kotlin SDK yet but even if it existed it still will use the web APIs under the hood similar to the [JS](https://github.com/pocketbase/js-sdk) and [Dart](https://github.com/pocketbase/dart-sdk) SDKs_).

**Reply by MHubertus** [reply]
Sorry, the question of course was if there are any sdks for Java or Kotlin.

**Reply by abbasalim** [reply]
https://github.com/agrevster/pocketbase-kotlin
